source/dotnet/Library/AdaptiveCards/SerializableDictionary.cs

```diff
@@ -0,0 +1,83 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Xml.Serialization;
+
+namespace AdaptiveCards
+{
+#if !NETSTANDARD1_3
+    [XmlRoot("dictionary")]
+    public class SerializableDictionary<TKey, TValue> : Dictionary<TKey, TValue>, IXmlSerializable
+    {
+        public SerializableDictionary()
```

Comment:
Thanks for doing this Tom; This is missing the Msft copyright header if you could add it in.. the newer guidelines need just a trimmed down version per [https://docs.opensource.microsoft.com/releasing/copyright-headers.html](https://docs.opensource.microsoft.com/releasing/copyright-headers.html)